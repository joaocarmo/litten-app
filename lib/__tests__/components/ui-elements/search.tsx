import TestRenderer from 'react-test-renderer'
import ThemeProvider from '@components/theme/provider'
import { UISearch } from '@ui-elements'

describe('Snapshot test for the "UISearch" component', () => {
  it('renders correctly', () => {
    const element = TestRenderer.create(
      <ThemeProvider>
        <UISearch placeholder="Search" />
      </ThemeProvider>,
    ).toJSON()
    expect(element).toMatchSnapshot()
  })
})
