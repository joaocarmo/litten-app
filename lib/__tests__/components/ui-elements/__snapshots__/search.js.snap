// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for the "UISearch" component renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "position": "relative",
      },
      undefined,
    ]
  }
>
  <TextInput
    autoCapitalize="none"
    enablesReturnKeyAutomatically={true}
    onChangeText={[Function]}
    onSubmitEditing={[Function]}
    placeholder="Search"
    placeholderTextColor="#c4c4c4"
    returnKeyType="search"
    style={
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 24,
          "minHeight": 44,
          "padding": 8,
          "paddingLeft": 24,
        },
        undefined,
      ]
    }
    value=""
  />
</View>
`;
